examples/plot_05_train_test_split.py

```diff
@@ -0,0 +1,180 @@
+"""
+.. _example_train_test_split:
+
+==========================
+Enhancing train-test split
+==========================
+
+This example illustrates the motivation and the use of skore's
+:func:`skore.train_test_split` to get assistance when developing your
+ML/DS projects.
+"""
+
+# %%
+# Creating and loading the skore project
+# ======================================
+#
+# We start by creating a temporary directory to store our project such that we can
+# easily clean it after executing this example. If you want to keep the project, you
+# have to skip this section.
```

Comment:
```
# easily clean it after executing this example. If you want to keep the project, you
# have to skip this section.
```
If you skip the section then there won't be a project to keep, right? I don't get it

Reply:
Created a new issue as it concerns the other examples at #779